.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  extends: ["plugin:@typescript-eslint/recommended"],
+  parserOptions: {
```

Comment:
Why doesn't this point to `tsconfig.json`?

Reply:
Good point - it does now.